wp-content/themes/access/src/js/modules/office-map.js

```diff
@@ -125,30 +125,39 @@ class OfficeMap {
 
     // Attach handler for the autocomplete search box. This updates the map
     // position and re-sorts locations around that position.
-    this._searchEl.addEventListener('keyup', (event) => {
+    this._searchEl.addEventListener('keyup', event => {
+      const NYCbounds = {
+        'south': 40.4773991,
+        'west': -74.25908989999999,
+        'north': 40.9175771,
+        'east': -73.7002721
+      };
```

Comment:
This constant should probably be a static constant declared at the bottom of the list;
```
OfficeMap.NYC_BOUNDS = { ...
```